Form-Controls/styles.css

```diff
@@ -0,0 +1,69 @@
+
+
+body { 
+    background-color: rgb(252, 231, 231);
+}
+form {
+    max-width: 400px;
+    margin: 20px auto;
+    padding: 20px;
+    background-color: white;
+    border-radius: 8px;
+    font-family: Arial, sans-serif;
+}
+h1 {
+    text-align: center;
+    color:black;
+    padding-top: 40px;
+
+}
+
+label {
+    font-size: 14px;
+    color:black;
+    margin-bottom: 5px;
+    display: block;
+}
+
+input[type="text"], input[type="email"], input[type="date"], select {
+    width: 100%;
+    padding: 10px;
+    margin-top: 5px;
+    margin-bottom: 15px;
+    border: 1px solid black;
+    border-radius: 4px;
+    box-sizing: border-box;
+    font-size: 14px;
+    background-color: white;
+}
+
+input[type="submit"] {
+    width: 100%;
+    background-color: green;
+    color: white;
+    padding: 10px 15px;
+    margin-top: 10px;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+    font-size: 16px;
+}
+
+input, select {
+    display: block;
+}
+
+input[type="text"]:focus, input[type="email"]:focus, input[type="date"]:focus, select:focus {
+    border-color: green;
+    outline: none;
+}
+
+    input[type="submit"] {
+        font-size: 14px;
+    }
+    h2{
+        text-align: center;
+    }
+    span{
+        color: green;
+    }
```

Comment:
Please, be consistent with the naming of your files and file structure. Do you see what  is not consistent here?

Cakes-Co/style.css
Form-Controls/styles.css
Wireframe/css/style.css